contacts_test.go

```diff
@@ -28,12 +28,6 @@ func TestContactActionsRouting(t *testing.T) {
 			responsePayload: "{}",
 			f:               func(c *WalletClient) error { return c.AcceptContact(context.Background(), fixtures.PaymailAddress) },
 		},
-		{
```

Comment:
Why is it deleted?

Reply:
Configuration for this test case looks like this: 
```
	{
			name:            "ConfirmContact",
			route:           "/contact/confirmed/",
			responsePayload: "{}",
			f: func(c *WalletClient) error {
				contact := models.Contact{PubKey: fixtures.PubKey}

				passcode, _ := c.GenerateTotpForContact(&contact, 30, 2)
				return c.ConfirmContact(context.Background(), &contact, passcode, 30, 2)
			},
		},
```
I bring it back but IMO it's not readable